build.sbt

```diff
@@ -1,5 +1,5 @@
-ThisBuild / scalaVersion       := "2.13.3"
-ThisBuild / crossScalaVersions := Seq("2.12.12", "2.13.3")
+ThisBuild / scalaVersion       := "2.13.4"
+ThisBuild / crossScalaVersions := Seq("2.12.13", "2.13.4")
```

Comment:
For the time being I think 2.12 upgrade (maybe also 2.13) need to be done separately as the scoverage plugin is not working with it.  Try downgrading that and see if the build passes (or the error changes).

See: https://github.com/scoverage/sbt-scoverage/issues/319

Reply:
Done! Thanks @sloshy